app/enterprise/0.34-x/kong-manager/configuration/authentication.md

```diff
@@ -121,11 +122,15 @@ admin_gui_auth_conf={                                     \
 }
 ```
 
-The values above can be replaced with their corresponding values for your 
-custom LDAP configuration:
+The values above can be replaced with their corresponding values for your custom
+LDAP configuration:
 
+  - `<ENTER_YOUR_BIND_DN_HERE>` - Your LDAP Bind DN (Distinguished Name)
+        * Used to perform LDAP search of user. This bind_dn should have 
+          permissions to search for the user being authenticated.
```

Comment:
Should we also have this description in the property reference? Right now it's just:
`The DN to bind to`

[L125](https://github.com/Kong/docs.konghq.com/blame/d0e39c30a846346d923742ba54451526c127a5f3/app/_hub/kong-inc/ldap-auth-advanced/0.33-x.md#L125)